### PR TITLE
Render all modules on the page

### PR DESCRIPTION
**Pull request by mikeglopez**
Combines all team members' modules on the page. For some reason, Travis suddenly had an issue with the version of node I was using, so I updated that to 10.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,14 +1,33 @@
 const express = require('express');
+const httpProxy = require('http-proxy');
 
+const apiProxy = httpProxy.createProxyServer();
 const app = express();
 const port = 3010;
 
+// app.set('trust proxy', true);
```
Comment (akcipriano) [inline server/index.js]: still need your commented out code?

File: server/index.js [hunk 2]
```diff
@@ -1,14 +1,33 @@
...
+
+app.get('/totalReviews', (req, res) => {
+  apiProxy.web(req, res, { target: 'http://localhost:3004' });
+});
+
+app.get('/totalReviews', (req, res) => {
+  apiProxy.web(req, res, { target: 'http://localhost:3004' });
+});
+
+app.get('/allHomes', (req, res) => {
```
Comment (akcipriano) [inline server/index.js]: oops... I should take this out of my module. I never ended up using it. Sorry!

**Review by akcipriano — APPROVED** [review]
Looks good!

File: server/index.js (hunk 1 above)
Comment (mikeglopez) [reply server/index.js]: Thanks, forgot about this! Removed

File: server/index.js (hunk 2 above)
Comment (mikeglopez) [reply server/index.js]: Thanks! I'll remove this too!

**Comment by mikeglopez** [comment]
Changes implemented, thanks!

**Pull request merged**